docs/src/main/asciidoc/openapi-swaggerui-guide.adoc

```diff
@@ -284,6 +284,88 @@ quarkus.smallrye-openapi.path=/swagger
 
 Hit `CTRL+C` to stop the application.
 
+== Loading OpenAPI Schama From Static Files
+
+Instead of dynamically creating OpenAPI schemas from annotation scanning, Quarkus also supports serving static OpenAPI documents.
+The static file to serve must be a valid document conforming to the https://swagger.io/docs/specification[OpenAPI specification].
+An OpenAPI document that conforms to the  OpenAPI Specification is itself a valid JSON object, that can be represented in `yaml` or `json` formats.
+
+To see this in action, we'll create a static OpenAPI document `META-INF/openapi.yaml` for our `/fruits` path.
+You can choose a different document path as indicated in the <<open-document-paths, document paths tip below>>
+
+[source, shell]
+----
+openapi: 3.0.1
+info:
+  title: Static OpenAPI document of fruits resource
+  description: Fruit resources Open API documentation
+  version: "1.0"
+
+servers:
+  - url: http://localhost:8080/openapi
+    description: Optional dev mode server description
+
+paths:
+  /fruits:
+    get:
+      responses:
+        200:
+          description: OK - fruits list
+          content:
+            application/json: {}
+    post:
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Fruit'
+      responses:
+        200:
+          description: new fruit resource created
+          content:
+            application/json: {}
+    delete:
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Fruit'
+      responses:
+        200:
+          description: OK - fruit resource deleted
+          content:
+            application/json: {}
+components:
+  schemas:
+    Fruit:
+      properties:
+        description:
+          type: string
+        name:
+          type: string
+----
+
+We'll also have to add the `mp.openapi.scan.disable=true` configuration to your `application.properties` file to disable the dynamic generation of OpenAPI document through annotation scanning.
```

Comment:
By default, it will combine in the OpenAPI document the static file and the model generated from your application endpoints code. You can however force Quarkus to only take your static openAPI document by using `mp.openapi.scan.disable=true`.

Reply:
The reason for the rephrase is that your sentences are extremely negative and feel like you are limiting options. Also We will have to seems like forced at gun point :)

Reply:
+1 with the rephrase. Should be good now. Tell me what you think.